### PR TITLE
[0.2.x] REP-64 Support for Dominion

### PR DESCRIPTION
**Pull request by paouelle**
#### What does this PR do?
Modifies build process to deploy security policy and features file as artifacts.
Adds support for Dominion to existing itests.
Updates the Jenkins file to indicate when the build starts and ends via comments and email.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@kcover 
@peterhuffer 
@clockard 

#### How should this be tested? (List steps with links to updated documentation)
- successful build

#### What are the relevant tickets?
Fixes #27

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by paouelle** [comment]
build now

File: distributions/test/itests/pom.xml [hunk 1]
```diff
@@ -12,46 +12,113 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.codice.test.thirdparty</groupId>
             <artifactId>rest-assured</artifactId>
-            <version>${ddf.version}</version>
+            <version>${codice-test.version}</version>
+            <scope>test</scope>
         </dependency>
+        <!--<dependency>-->
```
Comment (peterhuffer) [inline distributions/test/itests/pom.xml]: ✏️ Remove commented out code.

Comment (paouelle) [reply distributions/test/itests/pom.xml]: Oops! It ain't code 😝

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
Build now
Fixed codice-test to specify the version of bundle artifacts for which permissions should be granted. This should address the itest failure observed.

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
build now
Attempt after fix in codice test

**Comment by paouelle** [comment]
build now

**Pull request merged**